lib/deqm_test_kit/measure_availability.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module DEQMTestKit
+  # MeasureAvailability test group ensures selected measures are available on the fhir server
+  class MeasureAvailability < Inferno::TestGroup
+    id 'measure_availability'
+    title 'Measure Availability'
+    description 'Ensure selected measures are available on the fhir server'
+
+    fhir_client do
+      url :url
+    end
+
+    MEASURES = ['EXM130|7.3.000', 'EXM125|7.3.000'].freeze
```

Comment:
It doesn't look like the EXM125 module is ever used. Does it make sense to keep it in for future testing purposes? I guess the way we pull in measures will change soon (hopefully) regardless.

Reply:
Yeah- this is more of a temporary example to reflect how a set of measures will be used to populate the dropdown and then one of them will be selected by the user